### PR TITLE
Update turkish language

### PR DESCRIPTION
**Pull request by TA1GI**
Bugs fixed.
Missing messages added.

### CODE REVIEW

**Comment by thinkyhead** [comment]
Some strings may not be ideal because we are limited to 18 characters for menu titles and 20 characters for status messages. So please make sure that any updated strings are keeping to those limitations, even if they are not ideal.

File: Marlin/kaynak/lcd/dil/language_tr.h [hunk 1]
```diff
@@ -384,41 +562,51 @@ namespace Language_tr {
...
   LSTR MSG_ENDSTOP_ABORT                  = _UxGT("Endstop iptal");
   LSTR MSG_HEATING_FAILED_LCD             = _UxGT("Isınma başarısız");
-  LSTR MSG_ERR_REDUNDANT_TEMP             = _UxGT("Hata: Sıcaklık Aşımı");
+  LSTR MSG_ERR_REDUNDANT_TEMP             = _UxGT("Hata: SICAKLIK AŞIMI");
   LSTR MSG_THERMAL_RUNAWAY                = _UxGT("TERMAL PROBLEM");
+  LSTR MSG_TEMP_MALFUNCTION               = _UxGT("SICAKLIK ARIZASI");
   LSTR MSG_THERMAL_RUNAWAY_BED            = _UxGT("TABLA TERMAL PROBLEM");
-  LSTR MSG_THERMAL_RUNAWAY_CHAMBER        = _UxGT("ODA TERMAL PROBLEM");
+  LSTR MSG_THERMAL_RUNAWAY_CHAMBER        = _UxGT("KABİN TERMAL PROBLEM");
+  LSTR MSG_THERMAL_RUNAWAY_COOLER         = _UxGT("Soğutucu Termal Problem");
```
Comment (thinkyhead) [inline Marlin/kaynak/lcd/dil/language_tr.h]: For example, this string is 23 characters. It should be <= 20 characters.

File: Marlin/kaynak/lcd/dil/language_tr.h [hunk 2]
```diff
@@ -256,34 +345,58 @@ namespace Language_tr {
...
+  LSTR MSG_AMAX_B                         = _UxGT("Max ") STR_B _UxGT(" İvme");
+  LSTR MSG_AMAX_C                         = _UxGT("Max ") STR_C _UxGT(" İvme");
+  LSTR MSG_AMAX_N                         = _UxGT("Max @ İvme");
+  LSTR MSG_AMAX_E                         = _UxGT("Max E İvme");
+  LSTR MSG_AMAX_EN                        = _UxGT("Max * İvme");
+  LSTR MSG_A_RETRACT                      = _UxGT("G.Çekme Hızı");
+  LSTR MSG_A_TRAVEL                       = _UxGT("Gezinme Hızı");
+  LSTR MSG_INPUT_SHAPING                  = _UxGT("Girdi Şekillendirme");
+  LSTR MSG_SHAPING_ENABLE                 = _UxGT("@ şekillendirmesini etkinleştir");
+  LSTR MSG_SHAPING_DISABLE                = _UxGT("@ şekillendirmesini devre dışı bırak");
```
Comment (thinkyhead) [inline Marlin/kaynak/lcd/dil/language_tr.h]: Meanwhile, these strings are way too long. On character LCD screens users will only see this for both items:
```
@ şekillendirmesin
```

**Comment by TA1AUB** [comment]
What about now?

**Pull request merged**